integration_test/gce/gce_testing.go

```diff
@@ -411,18 +416,28 @@ func lookupTrace(ctx context.Context, logger *log.Logger, vm *VM, window time.Du
 	return traceClient.ListTraces(ctx, req)
 }
 
-// nonEmptySeries evaluates the given iterator, returning its first non-empty
-// time series. An error is returned if the evaluation fails.
-// A return value of (nil, nil) indicates that the evaluation succeeded
-// but returned no data.
-func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (*monitoringpb.TimeSeries, error) {
+// nonEmptySeriesList evaluates the given iterator, returning a non-empty array of
+// time series, the length of the array is guaranteed to be of size minimumRequiredSeries or greater.
+// A panic is issued if minimumRequiredSeries is zero or negative.
+// An error is returned if the iterator provides an array less than minimumRequiredSeries.
```

Comment:
We're still returning `(nil, nil)` in the case of a successful empty iterator, so I would reword this as:

```
An error is returned if the evaluation fails or produces a non-empty slice with length less than minimumRequiredSeries.
A return value of (nil, nil) indicates that the evaluation succeeded but returned no data.
```